packages/service/ni_measurement_plugin_sdk_service/_internal/parameter/metadata.py

```diff
@@ -58,6 +60,7 @@ class ParameterMetadata(NamedTuple):
 
     @staticmethod
     def initialize(
+        validate_type: bool,
```

Comment:
FYI: I don't like this parameter (too prominent, should not be necessary), but I see why you added it. It's okay for this PR.

https://github.com/ni/measurement-plugin-python/issues/824
https://github.com/ni/measurement-plugin-python/issues/825